src/lib/events/offlineRsvpSync.ts

```
@@ -0,0 +1,60 @@
+    const queue = await get<QueuedRsvp[]>(OUTBOX_KEY);
+    return queue || [];
+  } catch (error) {
+    console.error('[OfflineRsvpSync] Failed to get queued RSVPs', error);
+    return [];
+  }
+}
+
+export async function queueRsvpSubmission(payload: QueuedRsvp): Promise<void> {
+  try {
+    const queue = await getQueuedRsvps();
+    
```

ESLint

[error] Delete `····`

(prettier/prettier)